composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -207,7 +207,7 @@
 
     context "when the user is ignoring the latest version" do
       let(:ignored_versions) { [">= 1.22.0.a, < 3.0"] }
-      it { is_expected.to eq(Gem::Version.new("1.21.0")) }
+      it { is_expected.to eq(Gem::Version.new("2.8.0")) }
```

Comment:
These should have stayed at `"1.21.0"`, given that we're ignoring `[">= 1.22.0.a, < 3.0"]`... Fixing that in https://github.com/dependabot/dependabot-core/pull/6611